.github/workflows/pr.yml

```
@@ -74,10 +77,117 @@ jobs:
+        name: Package manager smoke (${{ matrix.package-manager }})
+        timeout-minutes: 10
+        runs-on: ubuntu-latest
+        permissions:
+            contents: read
+        strategy:
+            fail-fast: false
+            matrix:
+                package-manager: [npm, yarn, pnpm, bun]
+
+        steps:
+            -   name: Checkout repository
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)